soc/xtensa/esp32/soc.c

```diff
@@ -16,6 +16,28 @@
 #include <toolchain/gcc.h>
 #include <zephyr/types.h>
 
+#include <string.h>
+#include "esp_system.h"
+#include "esp_private/system_internal.h"
+#include "esp_attr.h"
+#include "esp_efuse.h"
+#include "esp_wifi.h"
```

Comment:
This list of includes seems too big. `string.h` is includes second time, `esp_efuse.h` and `esp_wifi.h` do not seem to be required. Please use minimal list of includes.